frontend/src/query_builder/GuiQueryEditor.jsx

```diff
@@ -59,8 +61,8 @@ export default class GuiQueryEditor extends Component {
         }
     };
 
-    setQuery(dataset_query) {
-        this.props.setQueryFn(dataset_query);
+    setQuery(query) {
+        this.props.setQueryFn(query);
```

Comment:
Sure. But why is it `dataset_query`, and not `datasetQuery`? It's weird having variables named with conventions from other languages